### ISSUE TITLE
adding mother

### ISSUE THREAD

**Issue by sevestar** [pawnesh/family_tree#2]
adding mother doesnot works.it doesnt display any node

**Comment by oldhobbyist** [comment]
+1

**Comment by devendratest** [comment]
Ye issue sahi kar do

**Comment by devendratest** [comment]
Kindly fix this issue.It is looking like incomplete.

**Comment by pawnesh** [comment]
Okay I will, may be this weak, thanks for pointing out.

**Comment by devendratest** [comment]
Hello I hope you have added..

**Comment by pawnesh** [comment]
I have added but still it has bug, I am working on it, right now mother will add but if father is not already exist, in case father exist mother is adding as grandmother.
working on this issue

**Comment by devendratest** [comment]
ok thanks

**Comment by msangapu** [comment]
Noticed this bug as well... seems like the logic needs improvement. I will see if I can fork this and fix the issue.

**Comment by pawnesh** [comment]
Yes I agree on the point of logic, check the **newway** folder, I am trying different way to do things. It is still in under construction.

**Comment by shrutiupari** [comment]
The send button not working, like entered data not getting sent. can you please guide me as its the first time I am working on family tree  as my task
